.travis.yml

```diff
@@ -5,11 +5,11 @@ services:
   - docker
 script:
   #- sudo PATH=$PATH GOPATH=$GOPATH make ci binaries
-  - make ci binaries
+  - make ci
```

Comment:
Please note that I'm not compiling, just doing the CI here. If you need to compile the `build` target can be added as dep to the CI target